_positions/product-manager.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Product Manager
+permalink: positions/product-manager/
+team: 18F
+active: true
+---
+**GS Grade Level:** 13
```

Comment:
@hestherchang The form currently requires the applicant to select either `GS 14` or `GS 15`. This listing shows only `GS 13`. Please clarify the intended GS levels.